src/modules/fw_autotune_attitude_control/fw_autotune_attitude_control.cpp

```diff
@@ -484,9 +484,9 @@ void FwAutotuneAttitudeControl::updateStateMachine(hrt_abstime now)
 	// In case of convergence timeout
 	// the identification sequence is aborted immediately
 	if (_state != state::wait_for_disarm && _state != state::idle && _state != state::fail && _state != state::complete) {
-		if (now - _state_start_time > 20_s
-		    || (_param_fw_at_man_aux.get() && !_aux_switch_en)
-		    || _start_flight_mode != _nav_state) {
+		if (now - _state_start_time > 20_s || (_param_fw_at_man_aux.get() && !_aux_switch_en)
```

Comment:
I think you can just remove the diff here

Reply:
I dont understand what you mean

Reply:
You just removed a newline here, right? I was probably not intended. Could you put it back so there is no difference between main and your branch in those lines?